app/main/routes_frontend.py

```diff
@@ -537,14 +541,24 @@ def accesspoint_credentials():
 def wireless_credentials():
     # TODO: properly handle No such file or directory by hiding settings in /setup or showing error
 
-    ssid = subprocess.check_output('more /etc/wpa_supplicant/wpa_supplicant-wlan0.conf | grep -w ssid | awk -F "=" \'{print $2}\'', shell=True)
-    psk = subprocess.check_output('more /etc/wpa_supplicant/wpa_supplicant-wlan0.conf | grep -w psk | awk -F "=" \'{print $2}\'', shell=True)
-    
+    # grep -w matches an exact word:
+    #   example non match:
+    #       echo "bssid=test-value" | grep -w ssid => ""
+    #   example match:
+    #       echo "bssid=test-value" | grep -w bssid => "bssid=test-value"
+    cmd_template = "more /etc/wpa_supplicant/wpa_supplicant-wlan0.conf | grep -v '^\s*[#]' | grep -w {key} "
```

Comment:
filtering out any line with comments... without this the commented out bssid is submitted to the server for storage if not deleted by the user